### DISCUSSION TITLE
How can I show co-pilot/codeium suggestions last?

### DISCUSSION THREAD

**Discussion by gabrielvincent** [saghen/blink.cmp#671] [General]
I'd like to always show LSP-related suggestions first. For example,

```javascript
const url = new URL("https://example.com");
url. // autocomplete here
```

When I hit `<leader>space` after `url.`, I'd like to see a list of the URL object properties appear first, and Codeium suggestions last.

**Comment by saghen** [comment]
Set `score_offset = some_negative_value` (i.e. -3) on the source

**Reply by gabrielvincent** [reply]
Where can I find in the documentation how to set the score offset of a source?

There's the default config:
``` lua
sources = {
  default = { 'lsp', 'path', 'snippets', 'buffer' },
  -- optionally disable cmdline completions
  -- cmdline = {},
},
-- allows extending the providers array elsewhere in your config
-- without having to redefine it
opts_extend = { "sources.default" }
```

How can I set the score_offset for a source?

**Reply by saghen** [reply]
You'll want to check the `default configuration` section

**Reply by gabrielvincent** [reply]
Thank you, I got it to work.
